README.md

```diff
@@ -1,2 +1,30 @@
-# OOP-school-library
-OOP
+# School Library
+
+> A OOP school library.
```

Comment:
- [ **OPTIONAL** ] As mentioned by the previous reviewer, I think it would be better if you update your project description to a meaningful one, telling what is about the preject itself.